tripal/expression/__init__.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+
+from tripal.client import Client
+
+logging.getLogger("requests").setLevel(logging.CRITICAL)
+log = logging.getLogger()
+
+
+class ExpressionClient(Client):
+    """Manage Tripal expressions"""
+
+    def add_expression(self, organism, analysis, feature_match, file_type, file_path,
+                       biomaterial_provider=None, array_design=None, assay_id=None,
+                       acquisition_id=None, quantification_id=None, file_extension=None,
+                       start_regex=None, stop_regex=None, no_wait=False):
+        """
+        :type organism: str
+        :param organism: Organism Id
+
+        :type analysis: str
+        :param analysis: Id of the analysis
+
+        :type feature_match: str
+        :param organism: Match to features using either name or unique_name
+
+        :type file_type: str
+        :param file_type: Expresssion file type : column or matrix
```

Comment:
Could it be guessed with python csv (or other)  module? (just counting the number of columns)